python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -158,33 +150,47 @@ async def begin_standby(self, id_data):
         await self.chillerModel.disconnect()
         self.kiloarcModel.disconnect()
```

Comment:
Please remove these two lines because they duplicate code in `handle_summary_state`. In handle_summary_state. you have:
```
if self.enabled_or_disabled:
    # ...
else:
     await self._common_close()
```
and that disconnects these two things.

Reply:
Also is it possible to be in the FAULT state when self.kiloarcModel or self.chillerModel are None? If so please update this code to handle that case. It's probably a good idea in any case.